source/changelogs/2020-12-16.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "2020-12-16"
+excerpt:
+category: 65ada3d7699b7900765d2ed5
```

Comment:
What does this refer to?

Reply:
Readme.com uses [yaml front matter](https://jekyllrb.com/docs/front-matter/) to push metadata along with the markdown files. The `category` identifies which section of docs where the markdown should go into. It's not required in the case of changelogs, so I'll remove these 👍